src/app/signOut/views/session-timeout.ejs

```diff
@@ -0,0 +1,21 @@
+<div class="govuk-width-container">
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+            <br>
+            <h2 class="govuk-heading-xl">
+                Sorry, your session has<br>timed out
+            </h2>
+
+            <h2 class="govuk-heading">Your session automatically ends if you don't do<br>anything for 20 minutes</h2>
```

Comment:
This class here is not recognised and makes the font look different. It should be replaced with class="govuk-heading-m"